core/src/com/unciv/logic/automation/civilization/DiplomacyAutomation.kt

```diff
@@ -125,7 +131,10 @@ object DiplomacyAutomation {
         // Don't accept if they are at war with our friends, they might use our land to attack them
         if (civInfo.diplomacy.values.any { it.isRelationshipLevelGE(RelationshipLevel.Friend) && it.otherCiv().isAtWarWith(otherCiv)})
             return false
-        if (hasAtLeastMotivationToAttack(civInfo, otherCiv, (diploManager.opinionOfOtherCiv()/ 2 - 10).toInt()) >= 0)
+        // Being able to see their cities can give us an advantage later on, especially with espionage enabled
+        if (otherCiv.cities.count { !it.getCenterTile().isVisible(civInfo) } < otherCiv.cities.count() * .8f)
+            return true
```

Comment:
I believe that the extra open borders motivation here increases the exploring ability by the AI a lot.